_about/feature-stages.md

```diff
@@ -75,7 +75,7 @@ Below is our list of existing features and their current phases. This informatio
 | [Service-to-service mutual TLS](https://istio.io/docs/concepts/security/mutual-tls.html)         | Beta
 | [Incremental Enablement of service-to-service mutual TLS](https://istio.io/docs/tasks/security/per-service-mtls.html)  | Alpha
 | [VM: Service Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)         | Alpha
-| [OPA Checker](https://github.com/mangchiandjjoe/istio/blob/d5390f6e436225949907d77ad3e9747a9bc26722/mixer/adapter/opa/README.md)  - **New to 0.4**      | Alpha
+| [OPA Checker](https://github.com/istio/istio/blob/41a8aa4f75f31bf0c1911d844a18da4cff8ac584/mixer/adapter/opa/README.md)  - **New to 0.4**      | Alpha
```

Comment:
potentially this could be "master" instead of a sha

Reply:
I'm not sure which is worse, the link falling out of sync, or dead links when the page inevitably moves.

Reply:
we detect 404s with the build so I think master is better